engines/director/channel.cpp

```diff
@@ -112,6 +112,12 @@ DirectorPlotData Channel::getPlotData() {
 	pd.dst = nullptr;
 
 	pd.srf = getSurface();
+	if (_sprite->_spriteType == kBitmapSprite &&
+		_sprite->_cast && _sprite->_cast->_type == kCastBitmap &&
+		((BitmapCastMember *)_sprite->_cast)->_bitsPerPixel == 1) {
```

Comment:
please check that `The Apartment` is still working, all three versions D2/D3/D4

Reply:
As far as I can tell everything there is rendering as normal.